epitome/dataset.py

```diff
@@ -87,9 +88,10 @@ def __init__(self,
           (ie. DNase, H3K27ac, etc.)
         '''
 
-
+        if assembly is not None:
+            self.assembly = assembly
```

Comment:
should assembly be set to a default here if it is None?

Reply:
If we set the default assembly in the constructor, then we can't print a warning in get_data_dir() that the assembly is None, and we will never be able to tell if only the data_dir is specified since the assembly will never be None.